recipe/meta.yaml

```diff
@@ -10,22 +10,34 @@ source:
   git_rev: v{{ version }}
   
 build:
-  number: 3
+  number: 4
   string: py_pb{{ protobuf | replace(".*", "")}}_{{ PKG_BUILDNUM }}
   noarch: python
+  ignore_run_exports:
+    - libgcc-ng
+    - libstdcxx-ng
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    # Use pins to control cos6/cos7 match
+    - libgcc-ng  {{ libgcc }}
+    - libstdcxx-ng  {{ libstdcxx }}    
   host:
     - bazel {{ bazel }}
     - python
+    # Use pins to control cos6/cos7 match
+    - libgcc-ng  {{ libgcc }}
+    - libstdcxx-ng  {{ libstdcxx }}
   run:
     - python 
     - absl-py {{ absl_py }}
     - googleapis-common-protos {{ googleapis_common_protos }}
     - protobuf {{ protobuf }}
+    # Use pins to control cos6/cos7 match
```

Comment:
This is the same as TensorFlow hub where a compiler is needed at build time, but no compiled code ends up in the package. So we don't want to add a run dep here on the libgcc libraries. And perhaps this PR is unnecessary.